### ISSUE TITLE
[Feature request] Index Management metrics (ISM & ILM)

### ISSUE THREAD

**Issue by sboulkour** [justwatchcom/elasticsearch_exporter#387]
The idea would be to gather metrics specific to Index Management operations, on some specific indices, 
- ISM being for Open Distro clusters
- ILM being for Elastic clusters.
  - Sample : https://www.elastic.co/guide/en/elasticsearch/reference/current/ilm-explain-lifecycle.html

**Comment by Shaybs** [comment]
Another request: The ability to get indices health statuses.

This was also needed by another user: https://stackoverflow.com/questions/60515645/how-to-get-elasticsearch-exporter-to-give-me-index-health-status

**Comment by ericandrewmeadows** [comment]
I agree - this would be nice as well.  I am putting together our monitoring currently for our ES cluster, and this item is missing.
